app/mailers/application_mailer.rb

```diff
@@ -4,6 +4,8 @@ class ApplicationMailer < ActionMailer::Base
 
   before_action :set_view_paths
 
+  track open: true, click: true
```

Comment:
For privacy friendliness, consider making these configurable settings and defaulting to `false`.